src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
```

Comment:
I noticed that you use inheritance for all the other subclasses and it's clean and easy to read for each individual subclass. Keep up the good work!